### PR TITLE
[FR] Update `Client/Keyboard_shortcuts`

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal to update the French translation of the wiki page `Shortcut_key_reference`.

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

*Feel free to merge if the delay is too long for a review by a native.*

### CODE REVIEW

File: wiki/Shortcut_key_reference/fr.md [hunk 1]
```diff
@@ -1,286 +1,295 @@
...
-- `F8`: Montre ou cache la fenêtre de Chat
-- `F9`: Montre ou cache la fenêtre de chat étendu
-- `F10`: Active ou désactive tous les boutons de la souris (sauf la molette de la souris)
-- `F12`: Prend une capture d'écran
-- `Shift` + `F12`: Prend une capture d'écran et la télécharge sur un serveur pour le partager
-- `Insert`: ([Boss Key](https://fr.wikipedia.org/wiki/Boss_key)) Réduit le client osu! dans la zone de notification
-- `Shift` + `F1`: Ouvre un [osu!status](https://twitter.com/osustatus) (quand une erreur de connection apparait)
+| Raccourci | Action |
+| :-- | :-- |
+| `Alt` + `F4` | Ferme le client (pas d'invite). Si vous êtes dans le gameplay, dans l'éditeur de beatmaps ou dans osu!direct, revient à la sélection des beatmaps ou au menu principal (pour osu!direct). |
```
Comment (Biokax) [inline wiki/Shortcut_key_reference/fr.md:23]:
```suggestion
| `Alt` + `F4` | Ferme le client (sans confirmation). Si vous êtes dans le gameplay, dans l'éditeur de beatmaps ou dans osu!direct, revient à la sélection des beatmaps ou au menu principal (pour osu!direct). |
```

File: wiki/Shortcut_key_reference/fr.md [hunk 2]
```diff
@@ -1,286 +1,295 @@
...
+| `Alt` + `Up/Down Arrow` | Règle le volume principal, utilisez les flèches gauche et droite pour régler les volumes des effets et de la musique. |
+| `Ctrl` + `Alt` + `Shift` + `S` | Recharge le skin actuel. Si vous appuyez sur cette touche pendant le jeu ou sur l'écran des résultats, elle sera exécutée après le retour à l'écran de sélection des beatmaps. |
```
Comment (Biokax) [inline wiki/Shortcut_key_reference/fr.md:25]:
```suggestion
| `Ctrl` + `Alt` + `Shift` + `S` | Recharge le skin actuel. Si vous appuyez sur cette touche en jeu, ou sur l'écran des résultats, elle sera exécutée après le retour à l'écran de sélection des beatmaps. |
```

File: wiki/Shortcut_key_reference/fr.md [hunk 3]
```diff
@@ -1,286 +1,295 @@
...
+# Référence des touches de raccourci
 
-Dans osu!, il existe divers raccourcis qui peuvent être utilisés dans le jeu et les menus. Ils sont listés ici avec leurs fonctions ci-dessous.
+Il existe de nombreux raccourcis dans osu! qui peuvent être utilisés dans le jeu ou en mode veille sur n'importe quel écran. Ils sont listés ci-dessous avec leurs fonctions.
 
-**Note:** quelques uns de ces raccourcis peuvent être changés dans les [options](/wiki/Options)! Toutefois, la liste ci-dessous présente les raccourcis par défaut.
+*Remarque : Cet article répertorie les raccourcis clavier par défaut. Vous pouvez modifier certains d'entre eux dans les [options](/wiki/Options).*
 
-## Jeux au sens large
+## Dans tout le jeu
```
Comment (Biokax) [inline wiki/Shortcut_key_reference/fr.md:17]:
```suggestion
## Général
```

File: wiki/Shortcut_key_reference/fr.md [hunk 4]
```diff
@@ -1,286 +1,295 @@
...
+| `Alt` + `Up/Down Arrow` | Règle le volume principal, utilisez les flèches gauche et droite pour régler les volumes des effets et de la musique. |
+| `Ctrl` + `Alt` + `Shift` + `S` | Recharge le skin actuel. Si vous appuyez sur cette touche pendant le jeu ou sur l'écran des résultats, elle sera exécutée après le retour à l'écran de sélection des beatmaps. |
+| `Ctrl` + `F11` | Active/Désactive le graphique des temps de trame. Appuyez sur `Ctrl` pour le mettre en pause. |
+| `F7` | Fait défiler les limiteurs de fréquence d'images. |
+| `F8` | Active/Désactive la fenêtre de tchat. |
+| `F9` | Active/Désactive la fenêtre de tchat étendue. |
+| `F10` | Active/Désactive tous les boutons de la souris (sauf la molette de la souris, qui est toujours activée). |
+| `F12` | Prend une capture d'écran. Cliquez sur la boîte de dialogue pour ouvrir le dossier des captures d'écran. |
+| `Shift` + `F12` | Prend une capture d'écran et téléchargez-la sur le serveur pour la partager. La capture d'écran sera également enregistrée localement. Cela ouvrira la capture d'écran sur votre navigateur web. |
```
Comment (Biokax) [inline wiki/Shortcut_key_reference/fr.md:32]:
```suggestion
| `Shift` + `F12` | Prend une capture d'écran et l'upload sur le serveur à des fins de partage. La capture d'écran sera également enregistrée localement. Cela ouvrira la capture d'écran sur votre navigateur web. |
```

File: wiki/Shortcut_key_reference/fr.md [hunk 5]
```diff
@@ -1,286 +1,295 @@
...
+| `Insert` | C'est la [boss key](https://fr.wikipedia.org/wiki/Boss_key). Réduisez le client de jeu dans la zone de notification. Cela rendra également le jeu muet. |
```
Comment (Biokax) [inline wiki/Shortcut_key_reference/fr.md:33]:
```suggestion
| `Insert` | C'est la [boss key](https://fr.wikipedia.org/wiki/Boss_key). Réduit le client dans la zone de notification. Cela rendra également le jeu muet. |
```

File: wiki/Shortcut_key_reference/fr.md [hunk 6]
```diff
@@ -1,286 +1,295 @@
...
+
+*Remarque : Toutes ces fonctions ne fonctionnent pas dans l'écran de sélection des beatmaps du mode Multi.*
+
+| Raccourci | Action |
+| :-- | :-- |
+| `F1` | Ouvre le menu de sélection des [modificateurs de jeu](/wiki/Game_modifier). |
+| `F2` | Sélectionne aléatoirement une beatmap. |
+| `Shift` + `F2` | Retourne à la beatmap précédemment sélectionnée de manière aléatoire. |
+| `F3` | Ouvre les options de la beatmap. |
+| `F5` | Retraite les beatmaps (cela peut vous demander de continuer). |
```
Comment (Biokax) [inline wiki/Shortcut_key_reference/fr.md:75]:
```suggestion
| `F5` | Recharger les beatmaps (demande une confirmation). |
```

File: wiki/Shortcut_key_reference/fr.md [hunk 7]
```diff
@@ -1,286 +1,295 @@
...
+| :-- | :-- |
+| `Alt` + `Enter` | Active/Désactive en mode plein écran. |
+| `Up/Down Arrow` | Règle le volume principal, utilisez les flèches gauche et droite pour régler les volumes des effets et de la musique. |
+| `Esc` | Revient en arrière ou annule. |
+| `P`, `E`, ou `O` | Ouvre le menu de lecture. |
+| `P` trois fois | Entre dans le mode Solo. |
+| `P` deux fois, puis `M` | Entre dans le mode [Multi](/wiki/Multi). |
+| `E` deux fois | Entre dans le mode Edit. |
+| `O` | Ouvre la barre latérale des options. |
+| `Q` | Affiche l'invite de sortie. |
```
Comment (Biokax) [inline wiki/Shortcut_key_reference/fr.md:49]:
```suggestion
| `Q` | Affiche le menu de sortie|
```

File: wiki/Shortcut_key_reference/fr.md [hunk 8]
```diff
@@ -1,286 +1,295 @@
...
+| `Esc` | Affiche l'invite de sortie, lorsque le menu d'options et de lecture est fermé. |
```
Comment (Biokax) [inline wiki/Shortcut_key_reference/fr.md:50]:
```suggestion
| `Esc` | Affiche le menu de sortie, lorsque le menu d'options et de lecture est fermé. |
```

File: wiki/Shortcut_key_reference/fr.md [hunk 9]
```diff
@@ -1,286 +1,295 @@
...
-- `H`: Cache ou décache le scoreboard et l'interface de replay (seulement disponible quand vous regardez un replay ou que vous êtes en mode spectateur)
-- `F3` or `F4`: Ajuste la vitesse pour for osu!mania (fonctionne seulement au début d'une partie)
-- `Ctrl` + `R` ou `` ` `` (maintenir brièvement): Réessayer rapidement la map en cours (Passe automatiquement l'intro quand vous la recommencez)
-
-## Editeur de Beatmap
-
-*Page principale: [Editeur de Beatmap](/wiki/Beatmap_Editor)*
+| Raccourci | Action |
+| :-- | :-- |
+| `Z` ou `F1` | Lit la musique précédente. |
```
Comment (Biokax) [inline wiki/Shortcut_key_reference/fr.md:57]:
```suggestion
| `Z` ou `F1` | Joue la musique précédente. |
```

File: wiki/Shortcut_key_reference/fr.md [hunk 10]
```diff
@@ -1,286 +1,295 @@
...
+| `X` ou `F2` | Joue la musique. Si elle est en cours, recommence la musique. |
+| `C` | Active/Désactive la pause et la lecture. |
+| `F4` | Arrête la lecture (réinitialise l'heure actuelle). |
```
Comment (Biokax) [inline wiki/Shortcut_key_reference/fr.md:60]:
```suggestion
| `F4` | Arrête la lecture (réinitialise le temps). |
```

File: wiki/Shortcut_key_reference/fr.md [hunk 11]
```diff
@@ -1,286 +1,295 @@
...
+| `V` ou `F5` | Joue la musique suivante. |
+| `J` ou `F6` | Ouvre le menu de saut vers. |
```
Comment (Biokax) [inline wiki/Shortcut_key_reference/fr.md:62]:
```suggestion
| `J` ou `F6` | Ouvre le menu de saut. |
```

File: wiki/Shortcut_key_reference/fr.md [hunk 12]
```diff
@@ -1,286 +1,295 @@
...
+| `V` deux fois | Cinema |
+| `B` | Score V2 |
+| `3` jusqu'à `9` | 3K jusqu'à 9K (osu!mania) |
+
+## Mode de lecture
+
+| Raccourci | Action |
+| :-- | :-- |
+| `+` ou `-` | Ajuste le décalage local (maintenez `Alt` pour changer la précision). |
+| `Esc` | Met le jeu en pause, ou revient à la sélection des beatmaps si vous regardez une rediffusion ou si le mode Auto ou Cinema est utilisé. |
```
Comment (Biokax) [inline wiki/Shortcut_key_reference/fr.md:126]:
```suggestion
| `Esc` | Met le jeu en pause, ou revient à la sélection des beatmaps si vous regardez une rediffusion ou si le mode Auto ou Cinéma est utilisé. |
```

File: wiki/Shortcut_key_reference/fr.md [hunk 13]
```diff
@@ -1,286 +1,295 @@
...
+| `Space` | Passe au gameplay. |
+| `Tab` | Active/Désactive le scoreboard. |
+| `Shift` + `Tab` | Active/Désactive le masquage de l'interface du jeu. |
```
Comment (Biokax) [inline wiki/Shortcut_key_reference/fr.md:129]:
```suggestion
| `Shift` + `Tab` | Active/Désactive l'interface du jeu. |
```

File: wiki/Shortcut_key_reference/fr.md [hunk 14]
```diff
@@ -1,286 +1,295 @@
...
+| `H` | Active/Désactive le masquage du tableau d'affichage et de l'interface du replay (lorsque vous regardez une replay ou que vous êtes spectateur). |
```
Comment (Biokax) [inline wiki/Shortcut_key_reference/fr.md:130]:
```suggestion
| `H` | Active/Désactive le tableau de score et l'interface du replay (lorsque vous regardez une replay ou que vous êtes spectateur). |
```

File: wiki/Shortcut_key_reference/fr.md [hunk 15]
```diff
@@ -1,286 +1,295 @@
...
+
+| Raccourci | Action |
+| :-- | :-- |
+| `Mouse Wheel Up/Down` | Recherche en avant/en arrière. |
+| `Shift` + `Mouse Wheel Up/Down` | Recherche en avant/en arrière à 4x. |
+| `Ctrl` + `Mouse Wheel Up/Down` | Ajuste le snap divisor. |
+| `Alt` + `Mouse Wheel Up/Down` | Ajuste le distance snapping multiplier lorsque vous passez le curseur sur le terrain de jeu. Ajuste le zoom de la timeline lorsque vous passez le curseur sur la timeline. |
+| `Alt` + `Shift` + `Mouse Wheel Up/Down` | Ajuste plus précisément le distance snapping multiplier lorsque vous survolez le terrain de jeu. |
+
+### Compose
```
Comment (Biokax) [inline wiki/Shortcut_key_reference/fr.md:193]:
```suggestion
### Composition
```

**Review by Biokax — CHANGES_REQUESTED** [review]

**Comment by Biokax** [comment]
A lot of things in that pr and others are like google translated, and that translated a lot a words like section titles that was good into something meanless. They are so much pr with problems that we don't have time to review all of the mess before merging with "no native". I think it will be better to focus on the quality of pr instead of quantity

**Comment by Walavouchey** [comment]
If you think a translation pr has many problems and need more time to review it, then tell us by commenting on that pr.

File: wiki/Shortcut_key_reference/fr.md (hunk 7 above)
Comment (DoryanR) [inline wiki/Shortcut_key_reference/fr.md:49]: | `Q` | Affiche le menu de sortie |

File: wiki/Shortcut_key_reference/fr.md [hunk 16]
```diff
@@ -1,286 +1,295 @@
...
-- `E`: Ajoute un son finish à/aux objet(s) sélectionnés(s)
-- `R`: Ajoute un son d'applaudissement à/aux objet(s) sélectionnés(s)
-- `Shift` + `Q`, `W`, `E`, `R`: Change le sampleset pour l'objet sélectionné en Auto, Normal, Soft ou Tambour respectivement
-- `Ctrl` + `Q`, `W`, `E`, `R`: Change le set additionnel pour l'objet sélectionné en Auto, Normal, Soft ou Tambour respectivement
-- `Ctrl` + `Shift` + `I`: Ouvre la fenêtre d'import de sample.
+| Raccourci | Action |
+| :-- | :-- |
+| `W` | Active/Désactive l'ajout du Whistle vers la sélection. |
+| `E` | Active/Désactive l'ajout du Finish vers la sélection. |
+| `R` | Active/Désactive l'ajout du Clap vers la sélection.. |
```
Comment (Biokax) [inline wiki/Shortcut_key_reference/fr.md:260]:
```suggestion
| `R` | Active/Désactive l'ajout du Clap vers la sélection. |
```

File: wiki/Shortcut_key_reference/fr.md [hunk 17]
```diff
@@ -1,286 +1,295 @@
...
 
-- `Shift` + `1`, `2`, `3`, `4`, `6`, `8`: Change le [Beat Snap Divisor](/wiki/Beatmap_Editor/Beat_Snap_Divisor) à 1/1, 1/2, 1/3, 1/4, 1/6, et 1/8 respectivement (1, 2, et 3 ne fonctionnent pas dans le mode d'édition osu!mania)
-- `Shift` Quand vous placez (timeline): Ignore le snapping de la grille et le beat (temps)
-- `J`: Cercle de décalage en arrière dans le temps
-- `K`: Cercle de décalage en avant dans le temps
-- `L`: Verrouille toutes les notes.
+| Raccourci | Action |
+| :-- | :-- |
+| `Shift` + (`1`, `2`, `3`, `4`, `5,`, `6`, `7`, `8`, or `9`) | Définit le [diviseur de mesure](/wiki/Beatmap_Editor/Beat_Snap_Divisor) à 1/1, 1/2, 1/3, 1/4, 1/5, 1/6, 1/7, 1/8 et 1/9 respectivement. |
+| `Shift` (tout en glissant dans la timeline) | Ignorez le snapping du diviseur de mesure. |
```
Comment (Biokax) [inline wiki/Shortcut_key_reference/fr.md:270]:
```suggestion
| `Shift` (tout en glissant dans la timeline) | Ignore le snapping du diviseur de mesure. |
```

File: wiki/Shortcut_key_reference/fr.md [hunk 18]
```diff
@@ -1,286 +1,295 @@
...
-- `Delete`: Supprime le layer sélectionné
-- `W`, `A`, `S`, `D`: Déplace le layer sélectionné vers le haut, la gauche, le bas ou la droite d'un osu!pixel
+| Raccourci | Action |
+| :-- | :-- |
+| `I` | Ouvre la superposition de la bibliothèque de sprites. |
+| `Ctrl` + `Z` | Annule. |
+| `Ctrl` + `Y` | Revient en arrière. |
+| `Ctrl` + `X` | Coupe le sprite sélectionné. |
+| `Ctrl` + `C` | Copie le sprite sélectionné. |
+| `Ctrl` + `V` | Colle le sprite à l'heure actuelle. *Remarque : les sprites collés n'apparaîtront pas dans la liste des calques tant que la beatmap n'aura pas été sauvegardé et rechargé. |
```
Comment (Biokax) [inline wiki/Shortcut_key_reference/fr.md:283]:
```suggestion
| `Ctrl` + `V` | Colle le sprite au temps actuel. *Remarque : les sprites collés n'apparaîtront pas dans la liste des calques tant que la beatmap n'aura pas été sauvegardé et rechargé. |
```

**Review by Biokax — CHANGES_REQUESTED** [review]

**Comment by TicClick** [comment]
`master` merged in to resolve the conflict, the review is still up to date

File: wiki/Client/Keyboard_shortcuts/fr.md [hunk 19]
```diff
@@ -1,286 +1,295 @@
...
-- `Ctrl` + `Mouse Wheel Up/Down`: Augmente/Réduit le diviseur de snap
-- `Alt` + `Mouse Wheel Up/Down`: Augmente/Réduit le multiplicateur de distance du alt-snapping quand vous êtes dans le terrain de jeux. Change le zoom de la timeline quand vous êtes dans la timeline
-- `Dragging`: Bouge une sélection de notes (la position et le temps), bouge les points dans un slider. Fait glisser la fin des sliders ou des spinners dans la timeline pour les étendre
+| Raccourci | Action |
+| :-- | :-- |
+| `T` | Définit le BPM et le décalage lorsque vous tapez sur le rythme de la chanson. |
+| `Shift` tout en ajustant le BPM ou le décalage | Incrémente 5 fois le montant normal. |
+| `Ctrl` tout en ajustant le BPM | Incrémente de 0,25. |
+| `Ctrl` tout en ajustant l'offset | Incrémente de 1. |
+| `Ctrl` tout en ajustant le slider velocity | Incrémente de 0.1. |
```
Comment (Walavouchey) [inline wiki/Client/Keyboard_shortcuts/fr.md:295]:
```suggestion
| `Ctrl` tout en ajustant le slider velocity | Incrémente de 0,1. |
```

File: wiki/Client/Keyboard_shortcuts/fr.md [hunk 20]
```diff
@@ -1,286 +1,295 @@
...
+  - keyboard
+  - shortcuts
+  - clavier
+  - raccourcis
 ---
 
 # Liste des raccourcis claviers
 
-Dans osu!, il existe divers raccourcis qui peuvent être utilisés dans le jeu et les menus. Ils sont listés ici avec leurs fonctions ci-dessous.
+Il existe de nombreux raccourcis dans osu! qui peuvent être utilisés dans le jeu ou en mode veille sur n'importe quel écran. Ils sont listés ci-dessous avec leurs fonctions.
```
Comment (Biokax) [inline wiki/Client/Keyboard_shortcuts/fr.md:13]:
```suggestion
Il existe de nombreux raccourcis dans osu! qui peuvent être utilisés en jeu ou depuis n'importe quel menu. Ils sont listés ci-dessous avec leurs fonctions.
```

File: wiki/Client/Keyboard_shortcuts/fr.md [hunk 21]
```diff
@@ -1,286 +1,295 @@
...
-- `F8`: Montre ou cache la fenêtre de Chat
-- `F9`: Montre ou cache la fenêtre de chat étendu
-- `F10`: Active ou désactive tous les boutons de la souris (sauf la molette de la souris)
-- `F12`: Prend une capture d'écran
-- `Shift` + `F12`: Prend une capture d'écran et la télécharge sur un serveur pour le partager
-- `Insert`: ([Boss Key](https://fr.wikipedia.org/wiki/Boss_key)) Réduit le client osu! dans la zone de notification
-- `Shift` + `F1`: Ouvre un [osu!status](https://twitter.com/osustatus) (quand une erreur de connection apparait)
+| Raccourci | Action |
+| :-- | :-- |
+| `Alt` + `F4` | Ferme le client (sans confirmation). Si vous êtes dans le gameplay, dans l'éditeur de beatmaps ou dans osu!direct, revient à la sélection des beatmaps ou au menu principal (pour osu!direct). |
```
Comment (Biokax) [inline wiki/Client/Keyboard_shortcuts/fr.md:23]:
```suggestion
| `Alt` + `F4` | Ferme le client (sans confirmation). Si vous êtes dans en jeu, dans l'éditeur de beatmaps ou dans osu!direct, revient à la sélection des beatmaps ou au menu principal (pour osu!direct). |
```

File: wiki/Client/Keyboard_shortcuts/fr.md [hunk 22]
```diff
@@ -1,286 +1,295 @@
...
-- `E`: Ajoute un son finish à/aux objet(s) sélectionnés(s)
-- `R`: Ajoute un son d'applaudissement à/aux objet(s) sélectionnés(s)
-- `Shift` + `Q`, `W`, `E`, `R`: Change le sampleset pour l'objet sélectionné en Auto, Normal, Soft ou Tambour respectivement
-- `Ctrl` + `Q`, `W`, `E`, `R`: Change le set additionnel pour l'objet sélectionné en Auto, Normal, Soft ou Tambour respectivement
-- `Ctrl` + `Shift` + `I`: Ouvre la fenêtre d'import de sample.
+| Raccourci | Action |
+| :-- | :-- |
+| `W` | Active/Désactive l'ajout du Whistle vers la sélection. |
+| `E` | Active/Désactive l'ajout du Finish vers la sélection. |
+| `R` | Active/Désactive l'ajout du Clap vers la sélection.. |
```
Comment (Biokax) [inline wiki/Client/Keyboard_shortcuts/fr.md:260]:
```suggestion
| `R` | Active/Désactive l'ajout du Clap vers la sélection. |
```

File: wiki/Client/Keyboard_shortcuts/fr.md [hunk 23]
```diff
@@ -1,286 +1,295 @@
...
 
-- `Shift` + `1`, `2`, `3`, `4`, `6`, `8`: Change le [Beat Snap Divisor](/wiki/Beatmap_Editor/Beat_Snap_Divisor) à 1/1, 1/2, 1/3, 1/4, 1/6, et 1/8 respectivement (1, 2, et 3 ne fonctionnent pas dans le mode d'édition osu!mania)
-- `Shift` Quand vous placez (timeline): Ignore le snapping de la grille et le beat (temps)
-- `J`: Cercle de décalage en arrière dans le temps
-- `K`: Cercle de décalage en avant dans le temps
-- `L`: Verrouille toutes les notes.
+| Raccourci | Action |
+| :-- | :-- |
+| `Shift` + (`1`, `2`, `3`, `4`, `5,`, `6`, `7`, `8`, or `9`) | Définit le [diviseur de mesure](/wiki/Beatmap_Editor/Beat_Snap_Divisor) à 1/1, 1/2, 1/3, 1/4, 1/5, 1/6, 1/7, 1/8 et 1/9 respectivement. |
+| `Shift` (tout en glissant dans la timeline) | Ignorez le snapping du diviseur de mesure. |
```
Comment (Biokax) [inline wiki/Client/Keyboard_shortcuts/fr.md:270]:
```suggestion
| `Shift` (tout en glissant dans la timeline) | Ignore le snapping du diviseur de mesure. |
```

File: wiki/Client/Keyboard_shortcuts/fr.md [hunk 24]
```diff
@@ -1,286 +1,295 @@
...
-- `Delete`: Supprime le layer sélectionné
-- `W`, `A`, `S`, `D`: Déplace le layer sélectionné vers le haut, la gauche, le bas ou la droite d'un osu!pixel
+| Raccourci | Action |
+| :-- | :-- |
+| `I` | Ouvre la superposition de la bibliothèque de sprites. |
+| `Ctrl` + `Z` | Annule. |
+| `Ctrl` + `Y` | Revient en arrière. |
+| `Ctrl` + `X` | Coupe le sprite sélectionné. |
+| `Ctrl` + `C` | Copie le sprite sélectionné. |
+| `Ctrl` + `V` | Colle le sprite à l'heure actuelle. *Remarque : les sprites collés n'apparaîtront pas dans la liste des calques tant que la beatmap n'aura pas été sauvegardé et rechargé. |
```
Comment (Biokax) [inline wiki/Client/Keyboard_shortcuts/fr.md:283]:
```suggestion
| `Ctrl` + `V` | Colle le sprite à l'horodatage actuel. *Remarque : les sprites collés n'apparaîtront pas dans la liste des calques tant que la beatmap n'aura pas été sauvegardé et rechargé. |
```

**Review by Biokax — CHANGES_REQUESTED** [review]

File: wiki/Client/Keyboard_shortcuts/fr.md [hunk 25]
```diff
@@ -20,7 +20,7 @@ Ces raccourcis fonctionnent partout :
 
 | Raccourci | Action |
 | :-- | :-- |
-| `Alt` + `F4` | Ferme le client (sans confirmation). Si vous êtes dans le gameplay, dans l'éditeur de beatmaps ou dans osu!direct, revient à la sélection des beatmaps ou au menu principal (pour osu!direct). |
+| `Alt` + `F4` | Ferme le client (sans confirmation). Si vous êtes dans en jeu, dans l'éditeur de beatmaps ou dans osu!direct, revient à la sélection des beatmaps ou au menu principal (pour osu!direct). |
```
Comment (Biokax) [inline wiki/Client/Keyboard_shortcuts/fr.md:23]:
```suggestion
| `Alt` + `F4` | Ferme le client (sans confirmation). Si vous êtes en jeu, dans l'éditeur de beatmaps ou dans osu!direct, revient à la sélection des beatmaps ou au menu principal (pour osu!direct). |
```
oops j'ai oublié celui là, désolé

**Review by Biokax — CHANGES_REQUESTED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**